lib/publiccloud/provider.pm

```diff
@@ -718,9 +718,7 @@ To get the complete output structure, the call is:
 
 sub get_terraform_output {
     my ($self, $jq_query) = @_;
-    assert_script_run('cd ~/terraform');
     my $res = script_output("terraform output -json | jq -r '$jq_query' 2>/dev/null", proceed_on_failure => 1);
```

Comment:
For `terraform output` to work, it has to be executed in the right current folder. So maybe it works (it work in VR http://autobot.qe.nue2.suse.org/tests/4376#step/ssh_interactive_end/35 ) but sound fragile.

From the point of view of mr_Test this VR solve the issue. mr_test and all tests using `qe-sap-deployment` does not care about this code.